src/views/AuthContainer.js

```diff
@@ -0,0 +1,23 @@
+define([
```

Comment:
nit: In new files you need to add the license header to the top (just copy it from another file)